src/Http/Http/src/Features/FormOptions.cs

```diff
@@ -16,83 +16,124 @@ public class FormOptions
         /// <summary>
         /// Default value for <see cref="MemoryBufferThreshold"/>.
         /// </summary>
+        /// <value>
```

Comment:
Do we know if `<value>` shows up in intellisense? Or only the summary?

Reply:
Looks like it doesn't show up, please move these values to the summary.

Reply:
The text is ok (Default , Defaults)? 
```
Default value for MemoryBufferThreshold. 
Defaults to 65,536 bytes‬, which is approximately 64KB.
```

Reply:
For these const values, lets have the format:

`Default value for MemoryBufferThreshold. 65,536 bytes‬, which is approximately 64KB.`

Reply:
Yes, that is good.